pkg/kubernetes/scripts/projects.js

```diff
@@ -103,6 +120,99 @@
         }
     ])
 
+
+    .controller('UserCtrl', [
+        '$scope',
+        '$routeParams',
+        '$location',
+        'kubeSelect',
+        'kubeLoader',
+        'projectData',
+        'projectActions',
+        'roleActions',
+        'ListingState',
+        function($scope, $routeParams, $location, select, loader, projectData, projectAction, roleActions, ListingState) {
+            loader.watch("users");
+            loader.watch("groups");
+            var user = $routeParams["user"] || "";
+            if (user) {
+
+                var userObj = select().kind("User").name(user);
+                if(!userObj || userObj.length < 1) {
+                    $location.path("/projects");
```

Comment:
We should be displaying a not-found pattern. @petervo Did you manage to do this for the volume objects?